util/lang-compress.pl

```diff
@@ -104,6 +96,10 @@
         }
         defined $1 ? $1 : " "
     }gsxe;
+
+    # Clean up spaces at start and end of lines
+    s/^ //mg;
+    s/ $//mg;
```

Comment:
I would actually use `\h+` instead of a mere space, just in case.

Reply:
Okay, the previous substitution is changing \h+ to space so it's kind of moot....

Reply:
Oh, right!

so, never mind...

(note to self: coffee first, comment later)

Reply:
Cool, I've reverted the change.